packages/commonwealth/server.ts

```diff
@@ -205,6 +205,10 @@ async function main() {
     app.use(passport.initialize());
     app.use(passport.session());
     app.use(prerenderNode.set('prerenderServiceUrl', 'http://localhost:3000'));
+    app.use(function (err, req, res, next) {
```

Comment:
Why are we adding this extra middleware? We already have an error handling middleware in `setupErrorHandlers`.

Reply:
removing this one! left as debugging statement - we already set this up in setupErrorHandlers!

Reply:
removed